gothemis/errors/error.go

```diff
@@ -1,18 +1,71 @@
+/*
+ * Copyright (c) 2016 Cossack Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Package errors describes errors returned by GoThemis functions.
 package errors
 
-// ThemisError is an operational error.
+/*
+#include <themis/themis.h>
+*/
+import "C"
+
+// ThemisErrorCode describes an error reported by Themis Core.
+type ThemisErrorCode int
+
+// Error code constants.
+//
+// These are numeric error codes used by Themis Core (see <themis/themis_error.h>).
+const (
+	Success          ThemisErrorCode = C.THEMIS_SUCCESS
```

Comment:
hmm, cgo knows about defines now? previously if I remember correctly, it didn't know about defines and we had should to copy-paste values to go code

Reply:
> hmm, cgo knows about defines now?

It does now. I thought that maybe this is some recent feature, but it seems not. GoThemis compiles fine with Go 1.8 and above, both before this PR and with these changes.